src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,77 @@
-import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    opacity: 1,
+  },
+  '50%': {
+    opacity: 0,
+  },
+  '100%': {
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: `${pixelsToRem(16)}rem`,
+      height: `${pixelsToRem(16)}rem`,
+      background: `radial-gradient(circle, transparent 53%, ${palette.LIVE_DARK} 53.5%, ${palette.LIVE_DARK} 54%)`,
+      marginInlineEnd: `${spacings.HALF}rem`,
+      position: 'relative',
+      transform: 'translate(0, 12%)',
+    }),
+
+  firstPromo: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_1_ONLY]: {
+        width: `${pixelsToRem(18)}rem`,
+        height: `${pixelsToRem(18)}rem`,
+      },
+      [mq.GROUP_2_ONLY]: {
+        width: `${pixelsToRem(24)}rem`,
+        height: `${pixelsToRem(24)}rem`,
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        width: `${pixelsToRem(28)}rem`,
+        height: `${pixelsToRem(28)}rem`,
+      },
+    }),
+
+  livelabelPulse: ({ palette }: Theme) =>
+    css({
+      '&:before': {
+        content: `''`,
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        borderRadius: '50%',
+        background: `radial-gradient(circle, ${palette.LIVE_DARK} 37%, ${palette.LIVE_DARK} 37.5%,transparent 38%)`,
+        [`@media (prefers-reduced-motion: no-preference)`]: {
```

Comment:
Does it need to be inside []? I think we only need [] for media queries so that JS can compute what the value should be
```suggestion
        '@media (prefers-reduced-motion: no-preference)': {
```

Reply:
Ah okay, I'd used an example I found in the repo which used brackets but it was using media queries of some sort...doh! 🥲 Have removed them and checked with the reduced motion toggled on. All good. 🙌